README.md

```diff
@@ -82,7 +90,8 @@ Sending an message returns the response body
 }
 ```
 
-## Send whatsApp template messages using the message builder please check our docs to see more information about whatsApp templates: https://docs.cmtelecom.com/en/api/business-messaging-api/1.0/index#whatsapp-template-message
+## Whatsapp Templates
+Send WhatsApp template messages using the message builder please check our docs to see more information about WhatsApp templates: https://docs.cmtelecom.com/en/api/business-messaging-api/1.0/index#whatsapp-template-message
```

Comment:
Can be replaced with:

https://developers.cm.com/messaging/docs/whatsapp#template

Reply:
```suggestion
Send WhatsApp template messages using the message builder please check our docs to see more information about WhatsApp templates: https://docs.cmtelecom.com/en/api/business-messaging-api/1.0/index#whatsapp-template-message
```

Reply:
Fixed!